webpack/webpack.min.config.js

```diff
@@ -13,7 +14,43 @@ module.exports = {
     devtool: false,
     mode: 'production',
     module: {
-        rules: [{ test: /\.glsl$/, use: 'webpack-glsl-loader' }]
+        rules: [
+            { test: /\.glsl$/, use: 'webpack-glsl-loader' },
+            { test: /\.svg$/, use: 'svg-inline-loader' }
+        ]
+    },
+    optimization: {
+        minimizer: [
+            new UglifyJSPlugin({
+                uglifyOptions: {
+                    output: {
+                        comments: false
+                    },
+                    compress: {
+                        unsafe_comps: true,
+                        properties: true,
+                        keep_fargs: false,
+                        pure_getters: true,
+                        collapse_vars: true,
+                        unsafe: true,
```

Comment:
`unsafe` doesn't feel very safe to me :slightly_smiling_face:

Reply:
Actually, by setting `unsafe:true` is not necessary to use `unsafe_comps:true`, it sets al `unsafe_` properties to `true`. We can discuss which ones we can set to true, or directly remove it (since it is false base default).

Here is the list https://github.com/mishoo/UglifyJS2/tree/harmony#compress-options

Reply:
I would remove all `unsafe` properties unless there is clear understanding of their behavior and if they bring an important measured improvement in size.

Reply:
I totally agree 👍 By removing them it goes from 241kb to 242kb, which for me is not very significant.